apps/api/scripts/test_tracked_todos.py

```
@@ -0,0 +1,926 @@
+import uuid
+from dataclasses import dataclass, field
+from datetime import datetime, timezone
+from pathlib import Path
+
+import httpx
+from jose import jwt
+
+# ── Config ─────────────────────────────────────────────────────────────────────
+
+API_BASE = "http://localhost:8000"
+AGENT_SECRET = "gaia_dev_testing_secret_2026"
```

Ruff (0.15.12)

[error] Possible hardcoded password assigned to: "AGENT_SECRET"

(S105)